samples/graph-teams-tag/nodejs/client/README.md

```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
```

Comment:
We already have a file outside in nodejs folder then why we need this ?

Reply:
No, removed